package.json

```diff
@@ -1,47 +1,47 @@
-{
-  "name": "gatsby-starter-default",
-  "description": "Gatsby default starter",
-  "version": "1.0.0",
-  "author": "Kyle Mathews <mathews.kyle@gmail.com>",
-  "dependencies": {
-    "emotion": "^9.2.10",
-    "gatsby": "^2.0.0",
-    "gatsby-plugin-i18n": "^0.4.2",
-    "gatsby-plugin-manifest": "^2.0.2",
-    "gatsby-plugin-offline": "^2.0.5",
-    "gatsby-plugin-react-helmet": "^3.0.0",
-    "gatsby-plugin-typescript": "^2.0.0",
-    "gatsby-source-filesystem": "^2.0.1",
-    "gatsby-transformer-remark": "^2.1.4",
-    "lodash": "^4.17.11",
-    "react": "^16.5.1",
-    "react-dom": "^16.5.1",
-    "react-helmet": "^5.2.0",
-    "ts-node": "^7.0.1",
-    "typescript": "^3.1.1"
-  },
-  "keywords": [
-    "gatsby"
-  ],
-  "license": "MIT",
-  "scripts": {
-    "build": "gatsby build",
-    "develop": "gatsby develop",
-    "lint": "tslint --project .",
-    "lint:fix": "tslint --project . --fix",
-    "format": "prettier --parser typescript --trailing-comma es5 --single-quote --write \"src/**/*.ts*\"",
-    "test": "echo \"Error: no test specified\" && exit 1",
-    "typecheck": "tsc -p . --skipLibCheck --noEmit"
-  },
-  "devDependencies": {
-    "@types/lodash": "^4.14.117",
-    "@types/react": "^16.4.18",
-    "@types/react-helmet": "^5.0.7",
-    "prettier": "^1.14.2",
-    "tslint": "^5.11.0"
-  },
-  "repository": {
-    "type": "git",
-    "url": "https://github.com/gatsbyjs/gatsby-starter-default"
-  }
-}
+{
```

Comment:
Probably line ending change (unforunately tslint only covers TypeScript files...)
I'd be happier with LF, but whatever